internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/model/job.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/definition/key"
+)
+
+// Job contains all Keboola-specific data we need for polling jobs.
+// At the end throttle the sink, so it does not overloads the service.
+type Job struct {
+	key.JobKey
+	Deleted bool   `json:"-"` // internal field to mark the entity for deletion, there is no soft delete
+	Token   string `json:"token"`
```

Comment:
I don't think Token should be copied here. You have the sink key meaning you can get retrieve the token for the sink when you need it.

```go
	// Get authorization token
	token, err := b.schema.Token().ForSink(file.SinkKey).GetOrErr(b.client).Do(ctx).ResultOrErr()
	if err != nil {
		return err
	}
```

Reply:
I use the token in `metacleanup.go`. I cannot access internal `schema` and I think it is easier to use single access to `job` instead of 2 calls to etcd because I did not store the Token within the `job`.

Reply:
Are you planning to make some changes regarding this or not? For the file upload I believe the credentials are actually different for each file and need to be as they're. This token is not different for each job, only for each sink.

Reply:
41c0053ff2be13d37daae46cd7cfbc793cf485b9